commit cb2a41457ecb6edad654135c8fdff62a560af1c3
Author: Matthias Reichl <hias@horus.com>
Date:   Fri Feb 2 20:30:42 2018 +0100

    ASoC: allo-piano-dac: fix S24_LE format
    
    Remove set_bclk_ratio call so 24-bit data is transmitted in
    24 bclk cycles.
    
    Also remove hw_params and ops as they are no longer needed.
    
    Signed-off-by: Matthias Reichl <hias@horus.com>

diff --git a/sound/soc/bcm/allo-piano-dac.c b/sound/soc/bcm/allo-piano-dac.c
index eaf50fb6dbca..cfe81c946271 100644
--- a/sound/soc/bcm/allo-piano-dac.c
+++ b/sound/soc/bcm/allo-piano-dac.c
@@ -42,23 +42,6 @@ static int snd_allo_piano_dac_init(struct snd_soc_pcm_runtime *rtd)
 	return 0;
 }
 
-static int snd_allo_piano_dac_hw_params(
-	struct snd_pcm_substream *substream, struct snd_pcm_hw_params *params)
-{
-	struct snd_soc_pcm_runtime *rtd = substream->private_data;
-	struct snd_soc_dai *cpu_dai = rtd->cpu_dai;
-
-	unsigned int sample_bits =
-		snd_pcm_format_physical_width(params_format(params));
-
-	return snd_soc_dai_set_bclk_ratio(cpu_dai, sample_bits * 2);
-}
-
-/* machine stream operations */
-static struct snd_soc_ops snd_allo_piano_dac_ops = {
-	.hw_params = snd_allo_piano_dac_hw_params,
-};
-
 static struct snd_soc_dai_link snd_allo_piano_dac_dai[] = {
 {
 	.name		= "Piano DAC",
@@ -70,7 +53,6 @@ static struct snd_soc_dai_link snd_allo_piano_dac_dai[] = {
 	.dai_fmt	= SND_SOC_DAIFMT_I2S |
 			  SND_SOC_DAIFMT_NB_NF |
 			  SND_SOC_DAIFMT_CBS_CFS,
-	.ops		= &snd_allo_piano_dac_ops,
 	.init		= snd_allo_piano_dac_init,
 },
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1190_cb2a41457ecb6edad654135c8fdff62a560af1c3.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1190_cb2a41457ecb6edad654135c8fdff62a560af1c3.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
